src/jbake/content/features.adoc

```diff
@@ -106,14 +106,22 @@ Example:
 
 [source,java]
 ----
+import java.util.Map;
+import static fj.P.*;
+import fj.data.Option;
+import fj.data.TreeMap;
+import fj.P2;
```

Comment:
💯  this is much easier to understand with these imports in here

Reply:
@sappharx While you are working through these, feel free to PR some documentation improvements (even if just adding the imports)